lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -1854,7 +1854,13 @@ void mme_app_handle_initial_context_setup_rsp_timer_expiry(
    * is active. If so,then abort the attach procedure and release the UE
    * context.
    */
-  ue_context_p->ue_context_rel_cause = S1AP_INITIAL_CONTEXT_SETUP_TMR_EXPRD;
+  ue_context_p->ue_context_rel_cause = S1AP_INITIAL_CONTEXT_SETUP_FAILED;
```

Comment:
why do we change the cause? isn't the release reason timer expiration?

Reply:
S1AP_INITIAL_CONTEXT_SETUP_TMR_EXPRD cause was introduced specifically at s1ap task to prevent sending of UE context release command. Now since we want to send ue context release command to enb, I have reused the existing cause S1AP_INITIAL_CONTEXT_SETUP_FAILED which allows sending of ue context release command. So we do not need S1AP_INITIAL_CONTEXT_SETUP_TMR_EXPRD cause anymore.